### PR TITLE
Scipy suggestions

### PR DESCRIPTION
**Pull request by mwcraig**
Some of the changes in here are just whitespace to make a diff so that I could comment on those slides on github.

Feel free to incorporate or not as you see fit!

Let me know if you want me to flesh out the last few slides more and/or takl about vpython in physics education (that would be a bit of a digression from your talk, though).

I like your progression from lower to higher level ways of doing 3D graphics, and the motivation for pyglet. It might be a bit long (though I'm assuming you aren't going to have people read through the PyOpenGL and pyglet example in detail :))

In the morning I'll pick out a physics-y example of vpython that can be dropped in too.

### CODE REVIEW

File: _presentations/scipy2016.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ Catherine Holloway (@femion, CatherineH)
 
 ## Processing
 
- - …created by Ben Fry and Casey Reas in 2001 from the MIT media lab
+ - …created by Ben Fry and Casey Reas in 2001 from the MIT  media lab
```
Comment (mwcraig) [inline _presentations/scipy2016.md:62]: I was unclear here...is processing the name of the package?

File: _presentations/scipy2016.md [hunk 2]
```diff
@@ -292,7 +293,7 @@ Also a python OpenGL binding, but simplifies windowing and multimedia
 ## A Helper for Pyglet
 
 - …Write geometric shape primitives (sphere, box, etc) as "*objects*"
-- …Write mathematical primitives (vector, vertex, etc) as "*utils*"
+- …Write mathematical primitives (vector, vertex, etc)  as "*utils*"
```
Comment (mwcraig) [inline _presentations/scipy2016.md:296]: Maybe add subpackage (i.e. _utils subpackage_)? When I first read this slide I though "object" mean Python object, and wasn't sure what utils meant.

**Comment by CatherineH** [comment]
The talk is 30 minutes including questions, and I talk pretty fast when nervous. Timing myself at home it takes about 21 minutes, so I'll probably have to add material rather than subtract it. I'm not going over any of what the OpenGL does.

File: _presentations/scipy2016.md (hunk 1 above)
Comment (CatherineH) [inline _presentations/scipy2016.md:62]: Processing is the name of the project, the language, and the IDE

File: _presentations/scipy2016.md [hunk 3]
```diff
@@ -441,6 +457,6 @@ or
 
 1. … Rename pyglet_helper to vpython-pyglet?
 3. … 3D Text
-2. … Make vpython-jupyter compatible with pyglet_helper
+2. … Make vpython-jupyter  compatible with pyglet_helper
```
Comment (mwcraig) [inline _presentations/scipy2016.md:460]: I'm pretty sure that won't happen...the glowscript API is the one that has essentially been adopted as the "official" one, and is baked into the latest version of the _Matter and Interactions_ textbook, which is the only major one I know of that uses vpython (it is wrtten by Bruce Sherwood and Ruth Chaaby). A more formal specification of an API, and a set of tests that check API implementation may be doable though.

**Comment by mwcraig** [comment]
OK, I'll flesh out a couple more things in the morning then!

**Pull request merged**